### PR TITLE
DOS4 declaration updates

### PR DESCRIPTION
**Pull request by katstevens**
https://trello.com/c/kTVN6PlQ/532-clone-dos4-content

The first of several PRs for this - content changes all supplied by Stefan but please look out for copy/paste errors, broken `href`s etc.

Copies the ModernSlavery section wholesale from G11 with no changes. We don't want to prefill this section from G11 as it includes a document upload.

One question has changed significantly (a previous 'pass' answer of `True` will now fail) - however this question is not prefilled from previous declarations, so we should be fine.

### CODE REVIEW

File: frameworks/digital-outcomes-and-specialists-4/questions/declaration/GAAR.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Why a tax return was incorrect
 question: >
-  Please confirm whether any of your organisation’s tax returns, submitted on or after 1 October 2012, have been found to be incorrect on or after 1 April 2013 because:
+  Have any tax returns submitted by your organisation on or after 1 October 2012  been found to be incorrect on or after 1 April 2013 for any of these reasons:
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-4/questions/declaration/GAAR.yml]: Double space
```suggestion
  Have any tax returns submitted by your organisation on or after 1 October 2012 been found to be incorrect on or after 1 April 2013 for any of these reasons:
```

File: frameworks/digital-outcomes-and-specialists-4/questions/declaration/mitigatingFactors3.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: Mitigating factors for Modern Slavery reporting requirements
+question: >
+  If your organisation does not meet the requirements, please explain why.
+type: textbox_large
+
+hidden: true
+validations:
+  - 
+    name: answer_required
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-4/questions/declaration/mitigatingFactors3.yml]: Is the line break here (after the dash) necessary? I find it a bit confusing

Comment (katstevens) [reply frameworks/digital-outcomes-and-specialists-4/questions/declaration/mitigatingFactors3.yml]: Not really - it's consistent with the other `mitigatingFactors` questions though. I'm trying not to change the existing questions too much.

Comment (lfdebrux) [reply frameworks/digital-outcomes-and-specialists-4/questions/declaration/mitigatingFactors3.yml]: Okay might as well stay consistent

**Comment by katstevens** [comment]
@lfdebrux Re: additional links. The content's been approved by CCS so I'm reluctant to add anything extra in.

File: frameworks/digital-outcomes-and-specialists-4/questions/declaration/GAAR.yml (hunk 1 above)
Comment (katstevens) [reply frameworks/digital-outcomes-and-specialists-4/questions/declaration/GAAR.yml]: Fixed!

**Pull request merged**